src/aics-image-viewer/components/AxisClipSliders/index.tsx

```diff
@@ -206,7 +207,7 @@ const AxisClipSliders: React.FC<AxisClipSlidersProps> = (props) => {
   };
 
   // Pause when view mode or volume size has changed
-  useEffect(() => props.playControls.pause(), [props.mode, ...Object.values(props.numSlices)]);
+  useEffect(() => props.playControls.pause(), [props.mode, props.imageName]);
```

Comment:
the comment still says "volume size".  how is imageName a proxy for that?

Reply:
I see - could it even be the actual Volume object ref too? that's ultimately the identity of the image

Reply:
...it could be 🤔